brave-core/src/test/java/com/github/kristofa/brave/BoundarySamplerTest.java

```diff
@@ -0,0 +1,15 @@
+package com.github.kristofa.brave;
+
+import org.assertj.core.data.Percentage;
+
+import static org.assertj.core.data.Percentage.withPercentage;
+
+public class BoundarySamplerTest extends SamplerTest {
+  @Override Sampler newSampler(float rate) {
+    return BoundarySampler.create(rate);
+  }
+
+  @Override Percentage expectedErrorRate() {
+    return withPercentage(10);
```

Comment:
note that usually this is good in 3-5% range over 100K ids. However, sometimes it isn't, so the threshold is high to not flake travis.